src/test/java/task12/Task.java

```diff
@@ -0,0 +1,35 @@
+package task12;
+
+import java.util.Random;
+import java.util.Scanner;
+public class Task {
+    static void task1(){
+        Random rand = new Random();
+        int[] mass = new int[10];
+        for (int i = 0; i <mass.length;i++){
+            mass[i] = rand.nextInt();
+            if (mass[i]<0) mass[i]= 0;
+            System.out.println(mass[i]);
```

Comment:
you can print your array out of the loop

Reply:
the local variable of the loop " i ", which is not visible outside the loop, so I decided to display an array inside the loop

Reply:
yeah, this way you print each element of an array separately, in case you would print once outside the loop, less memory would be used :)